IdentityCore/src/util/NSString+MSIDExtensions.m

```diff
@@ -227,6 +230,11 @@ + (NSString *)msidWWWFormURLEncodedStringFromDictionary:(NSDictionary *)dict
          {
              [encodedString appendFormat:@"=%@", encodedValue];
          }
+        
+         else if ((encodedValue && ![encodedValue isKindOfClass:[NSNull class]]) && !encodedValue.length)
```

Comment:
can you add a unit test?

Reply:
We already have a unit test for this - testmsidWWWFormURLEncodedStringFromDictionary_whenKeyWithEmptyValue_shouldReturnUrlEncodedStri